stream_data.py

```diff
@@ -0,0 +1,15 @@
+from gen_temperature_stream import *
+from gen_soc_stream import *
+
+def stream_data(number_of_streams):
+    for i in range(number_of_streams):
+        temperature_list = temperature_stream()
+        print(",".join([str(i) for i in temperature_list]))
```

Comment:
The console output formatting is confusing. Just by seeing the output one cannot make out, which is temperature and soc value. It could have been better if the out format (csv /json etc) contains only one temperature and soc in a single line as suggested by Sudeep during assignment explanation.